bootloader/src/pyi_global.c

```diff
@@ -60,6 +60,24 @@ char *saved_locale;
  */
 
 #if defined(_WIN32) && defined(WINDOWED)
+void
+show_message_box(
+    const char *msg,
+    const char *caption,
+    UINT uType)
+{
+    wchar_t wmsg[MBTXTLEN];
+    wchar_t wcaption[MBTXTLEN] = L"";
+    if (!pyi_win32_utils_from_utf8(wmsg, msg, MBTXTLEN)) {
+        MessageBoxA(NULL, msg, caption, MB_OK | uType);
+    }
+    else {
+        pyi_win32_utils_from_utf8(wcaption, caption, MBTXTLEN);
+        MessageBoxW(NULL, wmsg, wcaption, MB_OK | uType);
```

Comment:
> Currently, all caption parameters with calling show_message_box are written in hard-coded US-ASCII.

Okay then.

> It will be 'patterned' garbage. That would be the hint of what the real message is. I think this garbage is better than the messages which don't have the given string.

You are right, maybe add a comment to the code.

Reply:
Please add a short comment about the captions being ASCII, too.